openslides_backend/action/actions/organization/initial_import.py

```diff
@@ -84,6 +87,24 @@ def check_empty_datastore(self) -> None:
         ):
             raise ActionException("Datastore is not empty.")
 
+    def translate_organization_and_theme(self, data: Dict[str, Any]) -> None:
+        if "organization" in data and "1" in data["organization"]:
+            organization = data["organization"]["1"]
+            Translator.set_translation_language(organization.get("default_language"))
```

Comment:
```suggestion
            Translator.set_translation_language(organization["default_language"])
```
possible, if you set it as required

Reply:
Done.